README.md

```diff
@@ -1,19 +1,16 @@
 # Helm Chart for Harbor
 
-**Notes:** The master branch is in heavy development, please use the codes on other branch instead.
+**Notes:** The master branch is in heavy development, please use the codes on other branch instead. A high available solution for Harbor based on chart can be find [here](docs/High%20Availability.md).  
 
 ## Introduction
-
 This [Helm](https://github.com/kubernetes/helm) chart installs [Harbor](https://github.com/goharbor/harbor) in a Kubernetes cluster. Welcome to [contribute](CONTRIBUTING.md) to Helm Chart for Harbor.
 
 ## Prerequisites
-
-- Kubernetes cluster 1.10+ with Beta APIs enabled
-- Kubernetes Ingress Controller is enabled
+- Kubernetes cluster 1.10+
```

Comment:
Shall we document if user chooses to use external DB/Redis the PVC settings for DB/Redis will be ignored?